code-challenges.test.js

```diff
@@ -80,13 +159,63 @@ var mixedDataArray1 = [3, "yo!", 94, -9, false, 0, 18, "hola!"]
 
 // a) Create a test with expect statements for each of the variables provided.
 
-var vowelTester1 = "learn"
+//create a function that takes a string as argument
+//returns index of first vowel
+
+//newArray = string.split() to split the string into a new array
+//use foreach to run throught the vlues of the array and retrun the indexof(value)
+
+////????? i finally gave up i have spend most of my day trying to make this work. 
+
+//?????i can make it five me the location of the vowels at one point i also had it giving pushing the first vowel to the new array but i I counlnt get it give me the location of the index 
+
+//????the .forEach was my last ditch effort to make it work. 
+
+// describe("When firsVowlIndex gets called, return only the index of the first vowel in string")
+// const argument = vowlTester1
+// const actualResults = firstLett(argument)
+// expect(actualResults).teEqual("1")
+
+
+// const firstVowleIndex = (string) => {
+//     let newArray = string.split('');
+//     // let firstLett =[]
+//     let firstVowel = newArray.indexOf('a' || 'e' ||'i' || 'o'|| 'u' || 'y')
```

Comment:
Main question to ask yourself here is what arguments does `.indexOf()` require? `.indexOf()` only takes a string where as passing `'a' || 'e' ||'i' || 'o'|| 'u' || 'y'` will evaluate to a truthy/falsy value because of the use of `||`

Reply:
also playing with `.indexOf()`using different inputs and outputs would be a good way of understanding how to use the method for this problem